packages/modules/error-handling-react/tsconfig.json

```diff
@@ -1,11 +1,11 @@
 {
   "compilerOptions": {
-    "target":  "es2020",
+    "target": "es2017",
```

Comment:
are we still having issues supporting es2020?

Reply:
i just reverted this because we refactored this pkg out of the base viewer, so we dont use it anymore and i didnt want to make any new changes that we wouldnt have published. leaving the pkg as is as of our 2.x release

Reply:
ah right. Should've paid closer attention to which package